### ISSUE TITLE
Update NCIC codes

### ISSUE THREAD

**Issue by cdmgtri** [NIEM/NIEM-Releases#219]
Update NCIC codes for NIEM 5.1:  

- MAK – 129 new codes; 4 updated codes
- VMA – 142 new codes; 16 updated codes
- VMO – 34 new codes

**Comment by cdmgtri** [comment]
Updated in NIEM 5.1 beta 1.

**Comment by steve962** [comment]
With the 5.1rc1 VMO Codes table, there's no longer any model descriptions to go by, which is a major issue for those of us who define pick lists based on the XSD descriptions.   I understand the desire to switch back to the 4.2 code style for this for backwards compatibility (even though the mapping of make to model in 5.0 was actually quite useful...), but removing the description of which models each code represents is a lot worse.   Is there, or will there be, a new Type which will contain the full model description(s) for each of the codes?

**Comment by aprilm99** [comment]
I understand the concern – there was no good way to make everyone happy on this code. 

However we deferred to NCIC and that is what we decided on with them.

A question -- with the non-unique codes that where there,  how did they build a picklist before?

For the XSTF – I sent an email in August asking for feedback and received no response. While I did not state the descriptions were going away, I did ask for use cases and if a super code type was needed somewhere.
